tools/migration/main.go

```diff
@@ -124,6 +125,15 @@ func exitErr(errstr string) {
 	os.Exit(1)
 }
 
+// exitErrorCloseLogger closes the logger and calls exitError
+func exitErrCloseLogger(errstr string, logger *internal.Logger) {
+	err := logger.Close()
+	if err != nil {
+		errstr = fmt.Sprintf("%s. Error closing logfile when reporting this error %s", errstr, err.Error())
```

Comment:
It's a little awkward, but it's unlikely to ever see it.  I can't think of a better solution.